### PR TITLE
Fix gh-1159: Add 2017 to Teacher Registration

### LINKED ISSUE #1159: There's no 2017 in the teacher registration dropdown menu for birth year
**TL;DR:** We have 2017 as an option for scratchers signing up from the homepage, but not for teachers. We should be consistent.

If you look at the dropdown from the homepage versus from `/educators/register`, the year 2017 is not included in the latter:
screen shot 2017-01-30 at 4 52 22 pm

Found while testing https://github.com/LLK/scratchr2/pull/4448, but definitely not a blocker.

### PR DESCRIPTION
**Pull request by TheGrits**
Should fix #1159 

# Test cases:
- 2017 should now be an option
- 1916 should no longer be an option

### CODE REVIEW

File: src/components/registration/steps.jsx [hunk 1]
```diff
@@ -324,7 +324,7 @@ module.exports = {
         },
         getYearOptions: function () {
             return Array.apply(null, Array(100)).map(function (v, id) {
-                var year = 2016 - id;
+                var year = 2017 - id;
```
Comment (rschamp) [inline src/components/registration/steps.jsx]: We shouldn't have hardcoded the year like this — would you please update this to get the current year?

Comment (TheGrits) [inline src/components/registration/steps.jsx]: @rschamp 👍 Good thinking! I'm assuming we'd like to be using ReactJS to get the year, or does this typical JS solution work? http://stackoverflow.com/a/6002276/

Comment (rschamp) [inline src/components/registration/steps.jsx]: Yep that JS solution works!  React tries not to get involved with "business logic" like this, and  only provides tools for building the UI.

**Comment by rschamp** [comment]
Thanks @TheGrits!

**Comment by TheGrits** [comment]
@rschamp Anytime!

**Pull request merged**